dev/benchmarks/macrobenchmarks/test/animated_complex_image_filtered_perf_e2e.dart

```diff
@@ -0,0 +1,16 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:macrobenchmarks/common.dart';
+
+import 'util.dart';
+
+void main() {
+  macroPerfTestE2E(
+    'animated_complex_image_filtered_perf',
+    kAnimatedComplexImageFilteredPerfRouteName,
+    pageDelay: const Duration(seconds: 1),
+    duration: const Duration(seconds: 5),
```

Comment:
I've always felt more comfortable with 10s of run time, but I notice that the other animated_* benchmarks are all set for 5s. 10s may capture more opportunities for GC and other issues that might make an animation janky. Most of the benchmarks in this directory specify 10s.

Reply:
These are really just in the realm of "prove this optimization works" kind of benchmarks. Almost all of the improvements should come from the raster thread.

Reply:
But if the optimization induces more churn in memory it can jeopardize jank. An animation that runs at 60fps with a 2-frame hiccup ever couple of seconds is not necessarily better than an animation that runs at 30fps forever with no missed 30fps frames.

Reply:
Totally and I've bumped up the duration to 10seconds, but I'll make the rather strong statement that there is no actual trade off with the repaint boundary change - since we are already doing all of the raster caching already and just throwing it away needlessly.